### PR TITLE
fix: isPrimitive function repair

### PR DESCRIPTION
**Pull request by kmdrGroch**
should solve #1133

### CODE REVIEW

**Comment by kmdrGroch** [comment]
It was really hard @nicojs ;D I didn't expect there could be an issue in your function and that "null" is primitive. But now all should work and be clear :)

**Comment by kmdrGroch** [comment]
Waiting for some easy task to do :)

File: packages/stryker/test/helpers/producers.ts [hunk 1]
```diff
@@ -19,8 +19,20 @@ export function mock<T>(constructorFn: sinon.StubbableType<T>): Mock<T> {
...
 }
 
-function isPrimitive(value: any) {
-  return value !== 'object' && !Array.isArray(value);
+/**
+ * @description Checks if variable is a primitive
+ * @param value
+ */
+function isPrimitive(value: any): boolean {
+  const type: string = typeof value;
```
Comment (nicojs) [inline packages/stryker/test/helpers/producers.ts]: Explicit type here is not needed.

File: packages/stryker/test/helpers/producers.ts [hunk 2]
```diff
@@ -19,8 +19,20 @@ export function mock<T>(constructorFn: sinon.StubbableType<T>): Mock<T> {
...
+  if (
```
Comment (nicojs) [inline packages/stryker/test/helpers/producers.ts]: This implementation can be simplified: 

```ts
return ['string', 'undefined', 'symbol', 'boolean'].indexOf(type) > -1
    || (type === 'number' && !isNaN(value)
    || value === null)
```

**Comment by nicojs** [comment]
> Waiting for some easy task to do :)

I've added some hacktoberfest labels. Feel free to look around 👍

**Comment by kmdrGroch** [comment]
@nicojs and right now? (I have checked these issues, but I don't find anything easy enough at this moment.)

**Comment by nicojs** [comment]
LGTM. Merge when green

**Comment by kmdrGroch** [comment]
@nicojs GREEN!!! :D

**Pull request merged**